app/controllers/api/v1/authentication_controller.rb

```diff
@@ -67,20 +67,20 @@ class AuthenticationController < Api::V1::ApiController
       def send_pin
         @mobile = Mobile.new(params[:mobile])
         unless @mobile.valid?
-          return render_error(@mobile.errors.full_messages.join('. '))
+          return render_error(@mobile.errors.full_messages.join(". "))
         end
 
         @user = User.find_by_mobile(@mobile.mobile)
```

Comment:
shouldn't this now be `find_user_by_mobile_or_email` instead ?

Reply:
Nope... send_pin end point is used for stock, admin and donor and not for broswe.

Reply:
didn't know there were 2 endpoints, which one is the browse endpoint ?

Reply:
Browse uses sign_up as we send pin even if user is not already available in db and after that we allow user to fill account details. So that pin always needs to go through process thus sign_up endpoint.

Reply:
I see, it's a bit weird that there's more sign-in functionalities in the sign_up method than in the sign_in method. Not worth making any changes now though. It'll probably clear itself up as our scope expands : )

Reply:
Yup 👍